lib/preneeds/middleware/response/clean_response.rb

```diff
@@ -4,7 +4,7 @@ module Middleware
     module Response
       class CleanResponse < Faraday::Response::Middleware
```

Comment:
Can't we devise our own codes? - I remember someone saying there might be better errors on their part sometime in the future, but until then.